WebAppForSolocoProject/Controllers/HomeController.cs

```diff
@@ -25,26 +28,28 @@ public HomeController(IConfiguration configuration, OwnerData ownerData)
         [HttpGet]
         public IActionResult Create()
         {
-            var model = new HomeCreateVM();
-            model.ownersList = ownerData.GetOwners();
-            model.basePath = ConfigurationManager.AppSettings["basePath"].ToString();
+            var model = new HomeCreateVM()
+            {
+                Logs = new List<string>()
+            };
+            model.OwnersList = ownerData.GetOwners().OrderBy(o => o.Name);
+            model.FolderList = model.OwnersList.First().QualityFolders;
+            model.BasePath = ConfigurationManager.AppSettings["basePath"].ToString();
             return View(model);
         }
 
         [HttpPost]
-        [ValidateAntiForgeryToken]
         public IActionResult Create(HomeCreateVM model)
         {
-            model.ownersList = ownerData.GetOwners();
-            Owner owner = ownerData.GetOwners().FirstOrDefault(o => o.Name == model.SelectedOwner);
+            UpdateModel(model);
             if (ModelState.IsValid)
```

Comment:
Maybe you can use FluentValidation to check if model is valid?

Reply:
It could be used for any controller

Reply:
I removed check if model state is valid, because all single messeges should be send to logs.
Instead of that I check if base path is null and add messege to logs